src/pages/en/guides/migrate-to-astro.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migrate an existing site to Astro
+description: Some tips and tricks for converting your site to Astro.
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import CMSGuidesNav from '~/components/CMSGuidesNav.astro';
+i18nReady: true
+---
+
+SARAH ROUGH NOTES
+
+**Ready to convert your site to Astro?** Follow one of our guides to migrate from a different project/framework.
+
+## Migration Guides
+
+<CMSGuidesNav />
+
+Note that many of these pages are **stubs**: they're collections of resources waiting for your contribution!
+
+## Why migrate your site to Astro?
+
+Astro provides many benefits: performance, simplicity and several built-in and available integrations!
+
+Depending on your existing project, you may be able to bring some, or much of your existing files. However, Astro's project structure is likely to be different than the one you're using. So, there are some key concepts that will help you make the transition to Astro.
```

Comment:
The following suggestion was originally written by @sarah11918:

```suggestion
Migrating may be less work than you think! Depending on your existing project, you may be able to fit some, or many of your files directly into Astro's project structure. Much of your existing content may be reusable when converted to `.astro` components.
```

What do we think about something like this?